hudi-cli/src/test/java/org/apache/hudi/cli/commands/TestCommitsCommand.java

```diff
@@ -569,4 +574,64 @@ public void testSyncCommits(HoodieTableType tableType) throws Exception {
     String expected = String.format("Load sync state between %s and %s", tableName1, tableName2);
     assertEquals(expected, result.toString());
   }
+
+  @Test
+  public void testInflightCommand() throws Exception {
+    generateData();
+    // Generate instant times using HoodieInstantTimeGenerator
+    String oldInstantTime1 = HoodieInstantTimeGenerator.formatDate(
+        new Date(System.currentTimeMillis() - 1000 * 60 * 60)); // 60 mins ago
+    String oldInstantTime2 = HoodieInstantTimeGenerator.formatDate(
+        new Date(System.currentTimeMillis() - 1000 * 60 * 45)); // 45 mins ago
+    String oldInstantTime3 = HoodieInstantTimeGenerator.formatDate(
+        new Date(System.currentTimeMillis() - 1000 * 60 * 5));  // 5 mins ago
+
+    // Reload meta client to pick up new instants
+    metaClient = HoodieTableMetaClient.reload(HoodieCLI.getTableMetaClient());
+
+    // Create inflight commits
+    InstantFileNameGenerator v1InstantNameGenerator = new InstantFileNameGeneratorV1();
+    List<String> fileNames = Arrays.asList(
+        v1InstantNameGenerator.makeInflightCommitFileName(oldInstantTime1),
+        v1InstantNameGenerator.makeCommitFileName(
+            oldInstantTime2 + "_" + InProcessTimeGenerator.createNewInstantTime()),
+        v1InstantNameGenerator.makeInflightCommitFileName(oldInstantTime3));
+    fileNames.forEach(name -> {
+      try {
+        Path filePath = new Path(basePath() + "/" + HoodieTableMetaClient.METAFOLDER_NAME + "/" + name);
+        HoodieTestDataGenerator.createEmptyFile(basePath(), filePath, storageConf());
+      } catch (IOException ignored) {
+        // Exception ignored.
```

Comment:
Why do we ignore the exception here?

Reply:
Since the file creation is under the lambda function it except the IOException to be catched. Ideally we dont expect failure, so did not add any print statement.